src/pages/journal/journal.effects.ts

```diff
@@ -149,6 +167,11 @@ export class JournalEffects {
         return of();
       }
       const nextDay = DateTime.at(selectedDate).add(1, Duration.DAY).format('YYYY-MM-DD');
+      this.analytics.logEvent(AnalyticsEventCategories.JOURNAL, AnalyticsEvents.NAVIGATION, this.analytics.getDescriptiveDate(nextDay));
```

Comment:
One thing I'd say, we could probably hide the details of the analytics logging behind some abstraction to make the effect more succinct.